relay-server/src/actors/project.rs

```diff
@@ -220,7 +220,7 @@ pub struct ProjectState {
     #[serde(skip, default = "Instant::now")]
     pub last_fetch: Instant,
 
-    /// True if this project state was fetched but incompatible with this Relay.
+    /// True if this project state failed fetching or was incompatible with this Relay.
```

Comment:
What does "incompatible" mean? Reading the docstrings of `invalid`, it seems to refer to projects that could not be parsed; but I don't know what the scope of "parse" is, if it's only e.g. syntax issues or network issues while fetching. Specifically, is an expired project state invalid?

Also, the docstrings for `invalid` are updated here, could we also update them for `invalid()`?